appengine-standard-archetype/pom.xml

```diff
@@ -25,7 +25,7 @@ limitations under the License.
 
   <groupId>com.google.appengine.archetypes</groupId>
   <artifactId>appengine-standard-archetype</artifactId>
-  <version>2.1.0-${appengine.sdk.version}</version>
+  <version>0.1.0</version>
```

Comment:
It was never published.  I find presenting weird version numbers to user disorienting.